fitbenchmarking/fitting_benchmarking.py

```diff
@@ -232,10 +230,38 @@ def make_plots(prob, visuals_dir, best_fit, wks, previous_name, count, user_func
         visuals_dir = support_pages_dir
```

Comment:
This is because in this PR, nist pages still do not use VDpages and their figures are saved in `/results/nist/figures` rather than `/results/nist/tables/support_pages`. This will change in PR #76 .